pkg/serviceaccount/jwt.go

```diff
@@ -54,7 +54,7 @@ type TokenGenerator interface {
 // JWTTokenGenerator returns a TokenGenerator that generates signed JWT tokens, using the given privateKey.
 // privateKey is a PEM-encoded byte array of a private RSA key.
 // JWTTokenAuthenticator()
-func JWTTokenGenerator(iss string, privateKey interface{}) (TokenGenerator, error) {
+func JWTTokenGenerator(iss, keyId string, privateKey interface{}) (TokenGenerator, error) {
```

Comment:
Wouldn't keyId need to come from somewhere so that it can change?

Reply:
For file-based reads, this should be static.  I'm just setting it to an empty string to preserve existing behavior.